_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+- You have basic familiarity with <prereq?>
+- <Other prereq?>
+
+## Configuring admin settings
+
+ Access to alerting and anomaly detection using visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges. **Stack Management** is the place where you manage advanced settings. The setting is enabled by default and appears as a feature in the **Stack Management > Advanced Settings > Visualizations** window as shown below. If the setting is disabled, it does not appear in this window. The following image shows the enabled settings in the Advanced Settings window.
+
+ <insert UI>
+
+If you are an administrator, you can disable the settings at the cluster level through the `dashboards.yml` configuration file, for example:
+
+```bash
+<insert code sample>
+```
+
+## Creating alerting monitors
+
+By default, when you begin to create the alert monitor workflow with the Discover application, you are presented with a menu-driven interface. This interface provides you with a range of options displayed in full screen, pop up, pull down, or drop down. In the interface, you define the metrics to monitor, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met.
+
+Alerts and anomalies are visualized in line charts on a dashboard. To create an alerting monitor using the Discover application and sample data and to visualize the alerts on a dashboard, follow these steps:  
+
+1. From the OpenSearch Dashboards main menu, select **Discover**.
+1. From **<window>**, select the **<name>** sample dataset. You will see a pre-populated dashboard with line chart visualizations.
+1. Select the ellipsis icon from the **<name>** visualization pane, and then from the **Options** pop-up window, choose **Alerting** > **Add alerting monitor**.
+1. Select **Create new monitor**.
+1. From the **Add alerting monitor** window, define the metrics and set the thresholds under **Monitor details** and **Triggers**.
+1. Under **Monitor details**, select <option> from the dropdown menu.
+1. Under **Triggers**, select <option> from the dropdown menu, and then specify a name and severity level for the trigger.
+1. _Optional_: In **Notifications**, select <option> from the dropdown menu, and then customize the notification message and select the Notification channel.
+1. Choose **Create monitor**.
+
+Once you have created a new monitor, the monitor is added to the visualization, as shown in the following image.  
+
+<insert UI>
+
+## Adding an associated monitor
+
+You can add existing monitors to a visualization using the Discover application instead of Stack Management, giving you a single user interface where can add, view, and edit monitors and monitor details. Continuing with the visualization and dashboard in the preceding tutorial, follow these steps to associate an existing monitor to a visualization:
```

Comment:
Not sure if there's some confusion about Stack Management? That is used for high-level configurations of Dashboards. This new feature is introducing the first way that monitors can be associated with visualizations, so I think we can just focus on that.

Reply:
Revised